traceevents/src/test/java/com/tomtom/kotlin/traceevents/TracerTest.kt

```diff
@@ -558,11 +560,6 @@ internal class TracerTest {
         assertEquals("x=10", Tracer.convertToStringUsingRegistry(SomeClass()))
     }
 
-    @Test
-    fun `equals and hashCode`() {
-        EqualsVerifier.forClass(TraceEvent::class.java).verify()
-    }
-
```

Comment:
Are you confident it doesn't matter that we broke `equals`?

Reply:
My bad, I see it's moved

Reply:
Equals is not broken. EqualsVerifier detects that backing field for `by lazy` is not used in equals. Which should not be used in equals nor hashCode.